sonar-plugin-api/src/main/java/org/sonar/api/source/Symbol.java

```diff
@@ -19,6 +19,12 @@
  */
 package org.sonar.api.source;
 
+import org.sonar.api.batch.sensor.SensorContext;
+
+/**
+ * @deprecated since 5.6 use {@link SensorContext#newSymbolTable()}
+ */
+@Deprecated
 public interface Symbol {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1133)